### ISSUE TITLE
Information provided for audio snippets, and their formatting

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/itwewina#100]
Currently, the presentation of audio snippets is as follows:

![image](https://user-images.githubusercontent.com/15003407/50131537-d7e7d200-023f-11e9-978c-78f45d9f0749.png)

On a longer term, we might want to present (or not) the following information.

0. For reasons of limited space,  the string 'Listen' is probably unnecessary.

1. Word form: Since there can be multiple inflected word forms with recordings associated with a lexical entry, spelling out the inflected form makes perfect sense (as is done currently).

2a. Speaker identity: All but one speaker have consented to being identified by name. While we could use the speaker id code, I'd tent to provide the full speaker name (except for those who want to remain anonymous).

2b. Speaker identity -> pic/bio link: As discussed in another issue (UAlbertaALTLab/recording-validation-interface#35), the speaker name should link to a webpage with the picture and bio of the speaker. Currently, we have only pictures of most speakers, but that may do for a start.

3. On a longer term, indicating the speaker community makes sense, in particular as we will want to distinguish these. However, currently we only have speakers from one community, so indicating that seems perhaps less informative. Nevertheless, we can keep the community for now, and see what the community feedback is in this matter.

4. Speaker gender: using the astrological signs for indicating this seems a snaffy and concise way of presenting this information.

**Comment by eddieantonio** [comment]
> 4\. Speaker gender: using the astrological signs for indicating this seems a snaffy and concise way of presenting this information.

Haha! Yeah, I may not display that information at all on the page; instead let users infer it from the voice and the speaker's name.

**Comment by aarppe** [comment]
Concerning the above, I'd (a) add speaker names, i.e. 2a above (as the anonymous person is not among speakers on the recordings we have preprocessed), and (b) remove 'listen', i.e. 0 above, and (c) wait for what the community says about seeing Maskwacîs throughout (my hunch is that they will appreciate it), i.e. keep 3 above as is. As to four, I'd agree that with speaker names and voice gender would be obvious.

Once we have simple bio pages, with at least the best pictures and names of the speakers (I might create this somewhere on sapir - but it may potentially eventually be moved to a Maskwacîs-administered server) - then we can link from speaker names to the speaker pages.

If I know where the URL per speaker is specified (e.g. in the recording database, where it can be linked to other speaker information, unless we'd want to implement that within itwêwina [more trouble, I presume?]), I can edit that to point to wherever I create the first version of the speaker bio page.

**Comment by eddieantonio** [comment]
Now:

 - [x] Add speaker names
 - [x] Remove 'listen'
 - [x] Leave "Maskwacîs" (for now) (perhaps update recordings-validation-api for this?)
 - [x] Remove gender symbol

**Comment by aarppe** [comment]
The gender sign might be useful for the anonymous speaker (what we told the speakers was that in the case of anonymization only gender and community would be conveyed). But as there is only one anonymous speaker currently, maybe this does't matter that much.

**Comment by eddieantonio** [comment]
Blocked by: UAlbertaALTLab/recording-validation-interface#40
